nixos/tests/all-tests.nix

```
@@ -949,6 +949,7 @@ in {
   snapper = handleTest ./snapper.nix {};
   snipe-it = runTest ./web-apps/snipe-it.nix;
   soapui = handleTest ./soapui.nix {};
+  soarca = handleTest ./soarca.nix {};
```

```suggestion
  soarca = runTest ./soarca.nix;
```

See https://github.com/NixOS/nixpkgs/issues/386873